.github/workflows/cross.yml

```diff
@@ -7,7 +7,7 @@ permissions:
 
 jobs:
   crossarch-check:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Did you mean 24.04 ?

Reply:
Latest is now 24.04. This downgrades to 22.04 to make the tests pass. It's similar to build issues that we've seen in other repos: https://github.com/coreos/afterburn/issues/1156

I have not figured out where the failure is exactly.

Maybe we should run this cross-build on the new aarch64 runners as that should be faster as well.